README.md

```diff
@@ -109,10 +109,26 @@ The easiest way to install LocalStack is via `pip`:
 pip install localstack
 ```
 
+Installing the `awscli` for running the aws commands through `cli`
```

Comment:
Please remove this section - we already have a section `Accessing the infrastructure via CLI or code` further below.